src/transforms/helmert.jl

```diff
@@ -5,19 +5,37 @@
 """
     @helmert Datumₛ Datumₜ (δx=0.0, δy=0.0, δz=0.0, θx=0.0, θy=0.0, θz=0.0, s=0.0)
 
-Helmert transform with translation parameters `δx, δy, δz` in meters, 
-rotation parameters `θx, θy, θz` in arc seconds,
-and scale parameter `s` in ppm (parts per million).
+Helmert transform with translation parameters `δx`, `δy` and `δz` in meters,
+rotation parameters `θx`, `θy` and `θz` in arc seconds, and scale parameter
+`s` in ppm (parts per million).
 
-## References
+The EPSG database includes two rotation conventions:
 
-* Section 4.3.3 of EPSG Guidance Note 7-2: <https://epsg.org/guidance-notes.html>
+1. "Coordinate Frame Rotation"
+2. "Position Vector transformation"
+
+In the convention (1), the rotation parameters `θx`, `θy` and `θz` are copied
+without modification. Consider the NZGD49 <> WGSG84 transform as an example:
+
+https://epsg.org/transformation_1564/NZGD49-to-WGS-84-2.html
+
+The rotation parameters are given by `θx=-0.47`, `θy=0.1` and `θz=-1.024`.
+
+In the convention (2), the sign of the rotation parameters must be flipped.
+Consider the OSGB36 <> WGS84 transform as an example:
 
-### Notes
+https://epsg.org/transformation_1314/OSGB36-to-WGS-84-6.html
 
-The convention used for rotation is the Position Vector. 
-To set rotation parameters that use the Coordinate Frame
-convention, simply invert the sign of the parameters.
+The rotation parameters are given by `θx=-0.15`, `θy=-0.247` and `θz=-0.842`.
+
+It is worth noting that convention (1) is the most frequent convention in the
+EPSG database, and that convention (2) is the one adopted by the C PROJ library:
```

Comment:
PROJ supports both conventions, but I think this text implied that only the Position Vector convention is supported. Did I misunderstand?